CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Minor: Added the ability to open an entire tab as a popup. (#2288, #3082)
```

Comment:
As agreed with pajlada, we no longer put issue numbers into changelog entries, only PR ones.
```suggestion
- Minor: Added the ability to open an entire tab as a popup. (#3082)
```